lang/en/auth_saml2.php

```diff
@@ -108,6 +108,14 @@
 $string['manageidpsheading'] = 'Manage available IdPs';
 $string['mdlattr_help'] = 'Which Moodle user field should the IdP attribute be matched to?';
 $string['mdlattr'] = 'Mapping Moodle';
+$string['wantassertionssigned'] = 'Want Assertions Signed';
```

Comment:
Moodle uses sentence case - not camel case - also check the other strings like consumer services...

Reply:
done